src/parse/asp/builtins.go

```diff
@@ -213,6 +221,11 @@ func buildRule(s *scope, args []pyObject) pyObject {
 			s.Error("%v", err)
 		}
 	}
+	if s.state.IsPendingTarget(target.Label) {
```

Comment:
Is ActivateTarget idempotent? Is there any concern about time-of-check/time-of-use here? Should we also be checking whether `target.State() < core.Active` (alternatively, should/does ActivateTarget make that check and short circuit)?